templates_test/main_test/mssql_main.tpl

```diff
@@ -0,0 +1,127 @@
+type mssqlTester struct {
+	dbConn     *sql.DB
+	dbName     string
+	host       string
+	user       string
+	pass       string
+	sslmode    string
+	port       int
+	testDBName string
+}
+
+func init() {
+	dbMain = &mssqlTester{}
+}
+
+func (m *mssqlTester) setup() error {
+	var err error
+	m.dbName = viper.GetString("mssql.dbname")
+	m.host = viper.GetString("mssql.host")
+	m.user = viper.GetString("mssql.user")
+	m.pass = viper.GetString("mssql.pass")
+	m.port = viper.GetInt("mssql.port")
+	m.sslmode = viper.GetString("mssql.sslmode")
+	// Create a randomized db name.
+	m.testDBName = randomize.StableDBName(m.dbName)
+
+	if err = m.dropTestDB(); err != nil {
+		return err
+	}
+	if err = m.createTestDB(); err != nil {
+		return err
+	}
+
+	createCmd := exec.Command("sqlcmd", "-S", m.host, "-U", m.user, "-P", m.pass, "-d", m.testDBName)
+
+	f, err := os.Open("tables_schema.sql")
+
+	if err != nil {
+		return errors.Wrap(err, "failed to open tables_schema.sql file")
+	}
+
+	defer f.Close()
+
+	createCmd.Stdin = newFKeyDestroyer(rgxMSSQLkey, f)
+
+	if err = createCmd.Start(); err != nil {
+		return errors.Wrap(err, "failed to start sqlcmd command")
+	}
+
+	if err = createCmd.Wait(); err != nil {
+		fmt.Println(err)
+		return errors.Wrap(err, "failed to wait for sqlcmd command")
+	}
+
+	return nil
+}
+
+func (m *mssqlTester) sslMode(mode string) string {
+	switch mode {
+	case "true":
+		return "true"
+	case "false":
+		return "false"
+	default:
+		return "disable"
```

Comment:
Yes, according to the driver description:

"encrypt"
- disable - Data send between client and server is not encrypted.
- false - Data sent between client and server is not encrypted **beyond the login packet.** (Default)
- true - Data sent between client and server is encrypted.